src/components/LeaderboardBtn.js

```diff
@@ -0,0 +1,37 @@
+import styled from "styled-components";
+
+const Btn = styled.div`
```

Comment:
Maybe also put styles to another file? I'll make discussion at group and then file with some tips